cdk-testsuite/tests/openshiftSanity.py

```diff
@@ -35,39 +37,120 @@ def setUp(self):
            
     def test_python_project(self):
         '''
-        TBD
```

Comment:
If you are directly importing a user created package in a test case file then executing it through avocado command gives the ERROR : ImportError: No module named libraries. By using the import module (imp.load_source()) can easily execute the command without any error. 

It needs more discussion. If you have any suggestion of directly import feel free to comment